docs/getting_started/index.rst

```diff
@@ -1,115 +1,164 @@
 Getting Started
 ===============
 
-.. note:: This document assumes you have a `working Ansible
+.. Note:: This document assumes you have a `working Ansible
           installation`_. If you don't, install Ansible before
           continuing. This can be done simply by running ``pip install -r
           requirements.txt`` from the root of the project.
 
-The Mantl project uses Ansible to bring up nodes and clusters. This generally
-means that you need three things:
+          It also assumes you have a working Terraform installation which you
+          can download from `Terraform downloads`_.
```

Comment:
This is awkwardly joined - how about "…working Terraform installation. You can download Terraform from…"